Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -1682,6 +1702,18 @@ DECLARE_SOA_TABLE(Run2BCInfos, "AOD", "RUN2BCINFO", run2::EventCuts, //! Legacy
                   run2::V0TriggerChargeA, run2::V0TriggerChargeC);
 using Run2BCInfo = Run2BCInfos::iterator;
 
+DECLARE_SOA_TABLE(Run2V0otfs, "AOD", "RUN2V0OTF", //! Run 2 V0 on the fly table
```

Comment:
Are we sure about the lower case here?
I wonder if `Run2V0OTF` is better?
@ddobrigk any opinion?

Reply:
I agree that upper case "OTF" is better here. And as a weak preference, I would in fact also go for "OTFV0s" instead of "V0OTFs" (since the plural is for V0s and not "V0 on the flies" somehow :-) ) - the rest looks ok to me. Once merged it would also be good to have a simple QA task in place in O2Physics. Thanks a lot!